config/settings.yml

```diff
@@ -19,3 +19,10 @@ plugins:
     default: 1
     enum: "TrustLevelSetting"
     client: true
+  flag_posts_voting_comments_allowed_groups:
+    default: "11" # auto group trust_level_1
+    type: group_list
+    client: false
```

Comment:
I think this is the default value. Do we want to be explicit about it?

Reply:
Ah good point, I will just remove :+1: